src/Microsoft.AspNetCore.Hosting.Server.Abstractions/IServer.cs

```diff
@@ -19,7 +19,8 @@ public interface IServer : IDisposable
         /// <summary>
         /// Start the server with an HttpApplication.
         /// </summary>
-        /// <param name="application">An instance of <see cref="IHttpApplication"/>.</param>
+        /// <param name="application">An instance of <see cref="IHttpApplication{TContext}"/>.</param>
+        /// <typeparam name="TContext">The context associated with the HttpApplication.</typeparam>
```

Comment:
Yuck. I would just say "application" then (lowercase). Can you patch up those two comments as well? I'm guessing they're copied from Katana or something.